src/building/how-to-build-and-run.md

```diff
@@ -226,16 +226,17 @@ Once you've created a `bootstrap.toml`, you are now ready to run
 `x`. There are a lot of options here, but let's start with what is
 probably the best "go to" command for building a local compiler:
 
-```bash
-./x build library
+```console
+./x build rustc
 ```
```

Comment:
Remark: hm, I wonder if we should suggest `./x build library` for producing a stage 1 rustc/std toolchain instead. But anyway, not for this PR.